From 306fa62f1c222ad1781519cb42191eeeba12e97c Mon Sep 17 00:00:00 2001
From: Antonio Borneo <antonio.borneo@foss.st.com>
Date: Fri, 25 Apr 2025 11:20:53 +0200
Subject: [PATCH 1683/1695] core: plat-stm32mp2: enable CFG_STM32_IRQ_NOTIF

The driver is used for buttons connected to wake-up pins.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git adfc5a1850056d9ef992d2509dbfaf63b3ea3a04 core: plat-stm32mp2: enable CFG_STM32_IRQ_NOTIF]
Signed-off-by: Antonio Borneo <antonio.borneo@foss.st.com>
Change-Id: I806ad358b0eeca1bd67ddf5dbfe082574ef7163b
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/454081
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
---
 core/arch/arm/plat-stm32mp2/conf.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/core/arch/arm/plat-stm32mp2/conf.mk b/core/arch/arm/plat-stm32mp2/conf.mk
index 626213ec9..98e76f02c 100644
--- a/core/arch/arm/plat-stm32mp2/conf.mk
+++ b/core/arch/arm/plat-stm32mp2/conf.mk
@@ -176,6 +176,7 @@ CFG_STM32_HSEM ?= y
 CFG_STM32_I2C ?= y
 CFG_STM32_IAC ?= y
 CFG_STM32_IPCC ?= y
+CFG_STM32_IRQ_NOTIF ?= y
 CFG_STM32_IWDG ?= y
 CFG_STM32_LPTIMER ?= y
 CFG_STM32_OMM ?= y
-- 
2.47.2

